staging/src/k8s.io/apiserver/pkg/server/options/recommended.go

```diff
@@ -117,6 +121,14 @@ func (o *RecommendedOptions) ApplyTo(config *server.RecommendedConfig) error {
 	if err := o.EgressSelector.ApplyTo(&config.Config); err != nil {
 		return err
 	}
+	if feature.DefaultFeatureGate.Enabled(features.RequestManagement) {
+		config.FlowControl = utilflowcontrol.NewRequestManager(
+			config.SharedInformerFactory,
+			kubernetes.NewForConfigOrDie(config.ClientConfig).FlowcontrolV1alpha1(),
+			config.MaxRequestsInFlight+config.MaxMutatingRequestsInFlight,
+			config.RequestTimeout/4,
```

Comment:
comment explaining why this is right please.

Reply:
Right, like in the other place the same calculation occurs.